sources/osgShader/node/textures.glsl

```diff
@@ -1,15 +1,19 @@
+#pragma DECLARE_FUNCTION
 vec3 textureRGB(const in sampler2D tex, const in vec2 uv) {
     return texture2D(tex, uv.xy ).rgb;
```

Comment:
not very interesting, but just noted strange spacing here and below if you happen to pass by